tests/ui/traits/const-traits/auxiliary/staged-api.rs

```diff
@@ -9,6 +9,11 @@
 pub trait MyTrait {
     #[stable(feature = "rust1", since = "1.0.0")]
     fn func();
+
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
What happens with const stability annotations here? I don't think our const checking logic is prepared to deal with those, is it?

Reply:
It's an error b/c the function is not const. I think it's fine to remain an error, since I don't think the checking logic is set up to deal with per-method const stability either.

Reply:
And writing `const fn` in a trait is not yet possible?

Maybe we should have an assert somewhere to ensure that if anyone ever adds that, the consequences are thought-through properly.

Reply:
> And writing const fn in a trait is not yet possible?

Nope, not yet. I could add an assert tho that we never encounter both parent const stability + fn signature constness. Maybe a delayed bug bc you can erroneously write `const fn` tho.